src/pasttle/model.py

```diff
@@ -23,7 +23,7 @@ class Paste(Base):
 
     __tablename__ = 'paste'
 
-    id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
+    id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True, index=True, unique=True)
```

Comment:
Ok, about the fail, I checked and it's not functional test but something more personal, adapt the code to your own liking / constraints if it bugs you, I won't mind :)